lib-injection/auto_inject.rb

```diff
@@ -0,0 +1,84 @@
+return if ENV['skip_autoinject']
+
+begin
+  require 'open3'
+
+  _, status = Open3.capture2e({'skip_autoinject' => 'true'}, 'bundle show ddtrace')
+
+  if status.success?
+    puts '[DATADOG LIB INJECTION] ddtrace already installed... skipping auto-injection'
```

Comment:
In this file, there's a lot of `puts` for debugging. Should there be a way of disabling them? E.g. by setting `DD_TRACE_DEBUG` perhaps?

Also, consider writing to standard error, e.g. `STDERR.puts`, rather than to standard out, so as to not affect applications that are run with pipes or where the output is important.